data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
Well, anyway.  If you really think there's a point in application-specific theming (and that it's not implemented generically in GTK :]), I don't mind so much.  I just don't really like the loss of flexibility we had on version-specific files (as yeah, GTK 3.20 changed everything), but well, let's hope they didn't break anything else, and that there's nothing we might ever need on < 3.20 that break >= 3.20.